### DISCUSSION TITLE
is it possible create schema without primary key or unique

### DISCUSSION THREAD

**Discussion by reksaaa** [prisma/prisma#17025] [Q&A]
in this case i need to create backend api from existing database  and that database dont have id or unique field because for reporting
when i pull db using prisma db pull is already generated but that table have comment

/// The underlying table does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.

is it possible to use prisma client without change my database

**Comment by pantharshit00** [comment]
Hello @reksaaa,

That is not possible right now. Having a primary key or a unique field is a hard requirement for Prisma right now. You can follow this issue for any updates: https://github.com/prisma/prisma/issues/7944

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
